tests/data/almanack/entropy/add_data.py

```diff
@@ -25,29 +25,40 @@ def commit_changes(directory: str, message: str) -> None:
 def create_repositories(base_path: pathlib.Path) -> None:
     """
     Sets up Git repositories with baseline content and adds entropy.
+    Repository Structure:
+        High_Entropy
+            - high_entropy.md
+                - Baseline Content
+                - Added Entropy
+            - high_entropy2.md
+                -Baseline content
+        Low_Entropy
+            -Low_entropy.md
+                - Baseline content
+                - Added entropy
     """
-
-    # Create directories for high_entropy and low_entropy
+    # Create and initialize directories for high_entropy and low_entropy
     for dir_name in ["high_entropy", "low_entropy"]:
         repo_path = base_path / dir_name
-        pathlib.Path(repo_path).mkdir(parents=True, exist_ok=True)
+        repo_path.mkdir(parents=True, exist_ok=True)
         git.Repo.init(repo_path)
 
-        md_file = repo_path / f"{dir_name}.md"
-        # Add baseline content to Markdown files
-        baseline_text = """
-            Baseline content
-            """
-        with open(md_file, "w") as f:
-            f.write(baseline_text)
+        # Create a markdown file with baseline content
+        with open(repo_path / f"{dir_name}.md", "w") as f:
+            f.write("Baseline content")
+
+            # Add a second file in the high_entropy repository and commit baseline content
+        with open(base_path / "high_entropy/high_entropy2.md", "w") as f:
+            f.write("Baseline content")
 
-        # Commit baseline content
+        # Commit the initial baseline content
         commit_changes(repo_path, "Initial commit with baseline content")
 
     # Run the add_entropy.py module
     insert_entropy(base_path)
 
     # Commit changes after adding entropy
     for dir_name in ["high_entropy", "low_entropy"]:
-        repo_path = base_path / dir_name
-        commit_changes(repo_path, "Commit with added entropy")
+        with open(base_path / "high_entropy/high_entropy2.md", "w") as f:
+            f.write("List of Number: 1\n,2\n,3\n,4\n,5\n,6\n,7\n8\n,9\n")
```

Comment:
Consider the use of [`range`](https://docs.python.org/3/library/functions.html#func-range) and [`join`](https://docs.python.org/3/library/stdtypes.html#str.join) to help create a more programmatic string of numbers as you have here.

Reply:
Referenced in test dataset issue